man/systemd.special.xml

```diff
@@ -277,6 +278,16 @@
             </para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><filename>first-boot-complete.target</filename></term>
+          <listitem>
+            <para>This target is intended as a synchronization point for units that need to run once during
+              the first boot.  Only after all units ordered before this target have finished, will the
+              machine-id be committed to disk, marking the first boot as completed.  If the boot is aborted
+              at any time before that, the next boot will re-run any units with <varname>ConditionFirstBoot=yes</varname>.
+            </para>
```

Comment:
this should be a "passive" unit, as mentioned earlier. i.e. one that is pulled in by the stutff ordered before it, not pulled in by the stuff ordered after it. There's a special section in the manpage for these, further below "Special Passive System Units"

Reply:
Ah, I misunderstood your earlier comment regarding this.  This makes sense, I'll change it.